src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,29 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={cn(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': completed,
+      },
+    )}
+    >
+      <h2 className="TodoInfo__title">
+        {`${title}`}
```

Comment:
```suggestion
        {title}
```
title is a string already.